### PR TITLE
Pick up and use the macros from krun_linux_kernel 4.9.88 branch.

### PR DESCRIPTION
**Pull request by vext01**
Changes in light of the new `<krun-syscall.h>` header recently merged into the Krun Linux kernel.

Looks good?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ If you are unable to do this, you can set `NO_MSRS=1` in your Unix environment
 when building Krun (see later), but be aware that this degrades the quality of
 the resulting benchmarking numbers.
 
+This version of Krun supports versions >= 4.9.88 of the custom kernel. Any
```
Comment (ltratt) [inline README.md]: Can we rephrase this to something a bit clearer. Maybe something like "Krun works on any version of the kernel if `NO_MSRS=1` is set. If a suitable version of the Krun custom kernel (>= 4.9.88) is used, the MSRs are also available"?

Comment (vext01) [reply README.md]: That's much better. I did struggle a bit wording this (long day)...

Comment (vext01) [reply README.md]: Fixed

**Comment by vext01** [comment]
Shall I squash?

**Comment by ltratt** [comment]
Yep, please squash.

**Comment by vext01** [comment]
splat.
